src/components/ClientEvents.vue

```diff
@@ -1,6 +1,113 @@
 <template>
   <section class="fields">
-    <h1>Events</h1>
+    <h1>Client Milestones</h1>
+    <div>
+      <div class="columns">
+        <div class="column">
+          <div class="field">
+            <button class="primary" @click="$router.push('/events/add')">
```

Comment:
`/` at the beginning of a route means relative to root, so either remove it or write everything out like `/client/${$route.params.id}/events/add`.